static/app-strings.json

```diff
@@ -2246,5 +2246,11 @@
   "Attach images by pasting or drag-and-drop.": "Attach images by pasting or drag-and-drop.",
   "There was a network error, but the publish may have been completed. Wait a few minutes, then check your Uploads or Wallet page to confirm.": "There was a network error, but the publish may have been completed. Wait a few minutes, then check your Uploads or Wallet page to confirm.",
   "Sports": "Sports",
-  "--end--": "--end--"
+  "--end--": "--end--",
+  "Default Video Quality": "Default Video Quality",
+  "Set a default quality for video playback. If the default choice is not available, the next lowest will be used when playback starts.": "Set a default quality for video playback. If the default choice is not available, the next lowest will be used when playback starts.",
+  "Auto": "Auto",
+  "Original": "Original",
+  "Enable default quality setting": "Enable default quality setting",
+  "Auto (%quality%) --[Video quality popup. Long form.]--": "Auto (%quality%)"
```

Comment:
Put `Auto (%quality%) --[Video quality popup. Long form.]--` next to `Auto --[Video quality. Short form]--` if can.  It helps the translator because they go through it one-by-one, and the surrounding entries help to provide context.